services/workflows-service/src/webhooks/webhooks.controller.ts

```diff
@@ -0,0 +1,55 @@
+import * as common from '@nestjs/common';
+import * as swagger from '@nestjs/swagger';
+import * as nestAccessControl from 'nest-access-control';
+import * as errors from '../errors';
+import { Public } from '@/common/decorators/public.decorator';
+import { AmlWebhookInput } from './dtos/aml-webhook-input';
+import { IndividualAmlWebhookInput } from '@/webhooks/dtos/individual-aml-webhook-input';
+import { WebhooksService } from '@/webhooks/webhooks.service';
+
+const WEBHOOKS = {
```

Comment:
Adhere to existing enum conventions.
```suggestion
const Webhook = {
```